jelauria_ComputedColumns.sql

```diff
@@ -0,0 +1,44 @@
+-- Computed column calculating the number of current counselors by gender
+CREATE FUNCTION jelauria_CALC_NumCurrentCounselorsByGender(@PK INT)
+RETURNS INT
+AS
+BEGIN
+    DECLARE @RET int
+    SET @RET = (SELECT COUNT(tEP.EmpPosID)
```

Comment:
Should this be `COUNT(DISTINCT tEP.EmpPosID)`?

Reply:
done!